var/spack/repos/builtin/packages/py-perfdump/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPerfdump(CMakePackage):
+    """An MPI- and HDF5- enabled Python module to create PAPI dumps"""
+
+    homepage = "https://github.com/RECUP-DOE/pyperfdump/"
+    git = "https://github.com/RECUP-DOE/pyperfdump.git"
+    url = "https://github.com/RECUP-DOE/pyperfdump/archive/refs/tags/v1.1.tar.gz"
+
+    maintainers("chaseleif")
+
+    license("Apache-2.0 WITH LLVM-Exception", checked_by="chaseleif")
+
+    version("1.1", sha256="d2d96e2bd8ba2616ea4a44233ea240a529788390a5c22d35f9de79a22647370d")
+
+    variant("mpi", default=True, description="Use MPI")
```

Comment:
We often prefer (expensive) variants to be off. This ensures that users are consciously enabling them.
```suggestion
    variant("mpi", default=False, description="Use MPI")
```